libcloudforensics/gcp.py

```diff
@@ -523,8 +530,8 @@ def _ReadStartupScript():
       startup_script = os.environ.get('STARTUP_SCRIPT')
       if not startup_script:
         # Use the provided script
-        startup_script = os.path.dirname(os.path.realpath(__file__)) + \
-                         '/../scripts/startup.sh'
+        startup_script = os.path.dirname(os.path.dirname(os.path.realpath(
+            __file__))) + '/scripts/startup.sh'
```

Comment:
Ah yes, I guess I also should've mentioned to use [os.path.join()](https://docs.python.org/3.7/library/os.path.html#os.path.join) here as well.